package.json

```diff
@@ -6,12 +6,12 @@
   "main": "bin/citgm",
   "preferGlobal": true,
   "bin": {
-    "citgm": "./bin/citgm",
-    "citgm-all": "./bin/citgm-all"
+    "citgm": "./bin/citgm.js",
+    "citgm-all": "./bin/citgm-all.js"
   },
   "scripts": {
     "test": "npm run lint && npm run tap",
-    "tap": "TAP_TIMEOUT=90 tap test/test-*.js",
+    "tap": "TAP_TIMEOUT=90 tap \"test/**/**/test-*.js\"",
```

Comment:
My mistake, "" _does_ suppress globbing in bash. It doesn't suppress variable expansion, so I had wrongly assumed globs were treated the same.